micropub.php

```diff
@@ -349,7 +361,10 @@ private static function create( $user_id ) {
 		if ( $user_id ) {
 			$args['post_author'] = $user_id;
 		}
-		$args['post_status'] = MICROPUB_DRAFT_MODE ? 'draft' : 'publish';
```

Comment:
i think we actually need to keep the constant around for backward compatibility, and fall back to it if the micropub request doesn't set post status and the setting is unset. sorry for not mentioning that sooner!

Reply:
I can make the default of the option triggered by it.